packages/rs-drive-abci/src/execution/engine/run_block_proposal/mod.rs

```diff
@@ -0,0 +1,46 @@
+use crate::error::Error;
+use crate::platform_types::platform::Platform;
+use crate::platform_types::{block_execution_outcome, block_proposal};
+use crate::rpc::core::CoreRPCLike;
+use dpp::validation::ValidationResult;
+use drive::grovedb::Transaction;
+
+mod v0;
```

Comment:
The point in isolation (using crates for example) that you can develop them idependenty and yout have well-defined interface of intercommunication between features. In this case we can reusie previous released creates instead of duplicating code in drive abci